component/src/main/resources/installation-scripts/install.bat

```diff
@@ -17,19 +17,49 @@ rem specific language governing permissions and limitations
 rem under the License.
 rem
 
-SETLOCAL
-SET /P ballerina_home=Please enter Ballerina home:
+setlocal
 
-IF NOT EXIST "%ballerina_home%/bin/ballerina.bat" (
-    ECHO "Incorrect Ballerina Home provided!"
+IF NOT EXIST "%BALLERINA_HOME%/bin/ballerina.bat" (
+    SET /P ballerina_home=Couldn't find Ballerina home in your system!.Please enter Ballerina home:
+)
+
+IF NOT EXIST "%BALLERINA_HOME%/bin/ballerina.bat" (
+    ECHO "Incorrect Ballerina home provided!"
     GOTO :END
+) ELSE (
+    SET ballerina_home=%BALLERINA_HOME%
 )
 
 SET ballerina_lib_location=%ballerina_home%\bre\lib
 SET ballerina_balo_location=%ballerina_home%\lib\repo
 SET version=${project.version}
 SET module_name=mongodb
 
+for /R "%ballerina_lib_location%" %%f in (wso2-mongodb*) do (
+    SET filename=%%~nf
```

Comment:
Even though it is highly unlikely that there could be other jars with a similar name, I don't think we should match file names this way and delete.
If you are doing this, IMO, you should exactly match the name pattern. For example, the mongodb module name is of this pattern, "wso2-mongodb-module-x.x.x.jar". Here x should be a positive integer.